velox/exec/HashAggregation.cpp

```diff
@@ -113,6 +113,10 @@ void HashAggregation::initialize() {
 
 bool HashAggregation::abandonPartialAggregationEarly(int64_t numOutput) const {
   VELOX_CHECK(isPartialOutput_ && !isGlobal_);
+  if (groupingSet_->hasSpilled()) {
+    // Once spilling kicked in, disable the abandoning code path.
```

Comment:
Does this mean to simplify the getOutput processing? Could you please help to add a comment on why disable the abandoning code path?

Reply:
Spilling would invalidate the condition check for abandoning. Say spilled rows will not count to `numOutput` based on current way of computation. I'll add some comments.